src/pages/AppConfigure.vue

```diff
@@ -26,6 +26,9 @@ const currentTab: Ref<ConfigureTabType> = ref('general')
 const router = useRouter()
 const store = useStore()
 const toast = useToast()
+const route = useRoute()
+
+currentTab.value = String(route.name).replace('Settings', '').toLowerCase()
```

Comment:
`currentTab` ref has a value `general` when the ref is initialised, but here we need to change the value of `currentTab` depending on the route it is currently in ( `general`, `auth`, `store` and `access` ). All these routes have name like `GeneralSettings`, `AuthSettings`, etc. 

That's the reason I need to do this step. A visual example of this can be found in below video at 0:22, the screen is at `/app/configure/access` and access tab is selected ( i.e `currentTab = 'access'` ). On refresh, the screen comes back to `app/configure/access` because of refresh logic and we need this additional logic to select the tab, access tab ( i.e make `currentTab = 'access'` )

https://user-images.githubusercontent.com/24249988/174064567-6d570ec4-d43f-492b-b44e-cdfe5b307f82.mp4